### PR TITLE
Adding custom domains for SRVLS services

### PR DESCRIPTION
**Pull request by abrennan89**
CP to 4.5, 4.6

### CODE REVIEW

**Review by nak3 — APPROVED** [review]

**Comment by nak3** [comment]
Looks good to me! I did not find any problem. Thank you Ashleigh.

File: modules/serverless-service-mesh-resources.adoc [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    kind: Service
+    name: istio-ingressgateway
+----
+
+.. Apply the YAML file:
++
+
+[source,terminal]
+----
+$ oc apply -n istio-system -f <filename>
```
Comment (mgencur) [inline modules/serverless-service-mesh-resources.adoc:128]: Maybe I'm just missing the final step of this procedure. That should be sending a `curl` request to the given service, maybe?

Comment (mgencur) [reply modules/serverless-service-mesh-resources.adoc:128]: Oh. I see it in the rendered document. It's in a different file. Cool.

**Comment by mgencur** [comment]
QE Approved.

File: modules/serverless-service-mesh-resources.adoc (hunk 1 above)
Comment (abrennan89) [reply modules/serverless-service-mesh-resources.adoc:128]: I think this is covered here? https://customdomain--ocpdocs.netlify.app/openshift-enterprise/latest/serverless/networking/serverless-ossm-custom-domains.html#serverless-access-custom-domain_serverless-ossm-custom-domains

File: modules/knative-service-cluster-local.adoc [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
++
+
+[source,terminal]
+----
+$ oc label ksvc <service_name> serving.knative.dev/visibility=cluster-local
+----
+
+.Verification steps
+
+* Check that the URL for your service is now in the format `http://<service_name>.<namespace>.svc.cluster.local`, by entering the following command and reviewing the output:
```
Comment (adellape) [inline modules/knative-service-cluster-local.adoc:29]: You can put a `\` at the front of the example URL to cancel out creating the link in the rendered version:

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#example-urls

File: modules/serverless-service-mesh-resources.adoc [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+.. Apply the YAML file:
++
+
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
+. Create an Istio `ServiceEntry` object. This is required for {ServerlessProductName} since Kourier is outside of the service mesh.
```
Comment (adellape) [inline modules/serverless-service-mesh-resources.adoc:73]: IBM Style Guide suggests "because" instead of "since" for translation reasons.

**Review by adellape — COMMENTED** [review]
A couple small things, but overall LGTM after a squash-ening! :+1:

**Pull request merged**